### DISCUSSION TITLE
TUF A15 2023 hibernate time

### DISCUSSION THREAD

**Discussion by DHuiji** [seerge/g-helper#2131] [Q&A]
I would like to ask you guys.

I set it in g-helper to go to hibernate after 180 minutes on "extra". However it went into hibernation within an hour, I've checked in the Windows Event Viewer. What can I do to make it hibernate at the time I set?

There is another question that should not be related to g-helper and I hope to get an answer here. The power light on this computer is supposed to flash when it is sleeping (at least it was when I bought it), but now it stays on and I can’t tell whether it is sleeping or not. Could it be related to Windows Modern Standby that i'm using?

Thank you for making this powerful software, the main functions are perfect. These are just some of the minor issues I encountered

**Comment by seerge** [comment]
@DHuiji windows can also hibernate device based on battery drain or some other params. I don't know if it can be controlled somehow. This is **separate** from a setting that you can set in G-Helper (which in turn is just a shortcut to corresponding setting in power plan).

As for keyboard animation during sleep. Your device is probably not properly sleeping (at least windows didn't communicate about that with hardware), so keyboard stays on.

You can try to investigate very bugged windows sleep with https://learn.microsoft.com/en-us/windows-hardware/design/device-experiences/modern-standby-sleepstudy . What often helps (in my experience) is to go to ``Windows -> Security & Maintenance`` app and manually run all ``Maintenance`` there.

**Reply by DHuiji** [reply]
Thank you very much for your reply. I've investigated it for a while and for future people.

Automatic hibernation is indeed a mechanism of Windows called "adaptive hibernate" and many posts say can be change by this command the percentage of battery drained before hibernating
powercfg /setdcvalueindex scheme_current sub_presence standbybudgetpercent 10 //10 means battery percentage

About me second question, i mean the single led that show the status, according to asus it flashes mean sleep and static mean power on. On windows sleepstudy-report I have verified that the laptop was sleeping(on S0) when the LED was static.
Keep investigating....
